backend/api-processor/workshop_processor.py

```diff
@@ -0,0 +1,60 @@
+#!/bin/python
```

Comment:
You should remove this file, we won't be using it, so should not be part of the pull request.